### PR TITLE
35 fixing versions

### PR DESCRIPTION
**Pull request by tschm**

### CODE REVIEW

**Comment by tschm** [comment]
I think fixing the versions will help us to avoid minor problems. Every update will be suggested and tested by dependabot

**Comment by phschiele** [comment]
> I think fixing the versions will help us to avoid minor problems. Every update will be suggested and tested by dependabot

Fine with me. I think it will make sense to add somewhere which versions exactly we used for the experiments in the paper for replication purposes.

File: README.md [hunk 1]
```diff
@@ -5,20 +5,13 @@ It contains a reference implementation of the Markowitz portfolio optimization
...
 ## Experiments
 
 Please run all experiments using
 
 ```bash
 make experiments
 ```
+
+Before the experiments start the virtual environment defined in 'requirements.txt'
+is replicated locally.
```
Comment (phschiele) [inline README.md:17]:
```suggestion
Before running the experiments, install the dependencies listed in `requirements.txt`.
```

**Comment by tschm** [comment]
I would add tag the repo and have a special tag for the one used for the paper

**Review by phschiele — APPROVED** [review]
See one suggestion above.

File: README.md (hunk 1 above)
Comment (tschm) [reply README.md:17]: The dependencies are installed automatically, no action is required

**Pull request merged**